src/Harvester/Harvester.cs

```diff
@@ -786,7 +786,17 @@ private List<BaseLogEntry> CheckForMissingFontErrors(string bookPath, Book book)
 			var harvestLogEntries = new List<BaseLogEntry>();
 
 			var missingFontsForCurrBook = GetMissingFonts(bookPath);
-			bool areAnyFontsMissing = missingFontsForCurrBook != null && missingFontsForCurrBook.Any();
+
+			if (missingFontsForCurrBook == null)
```

Comment:
This seems backwards somehow. If `missingFontsforCurrBook` is null, shouldn't that mean there aren't any fonts missing? What does GetMissingfonts do if everything is in order and all required fonts exist?